frontend/src/api/geo_location/transformer.js

```diff
@@ -4,7 +4,7 @@ const transformer = (item) => {
         'all_visitors_count': item.all_visitors_count,
         'new_visitors_count': item.new_visitors_count,
         'sessions_count': item.sessions_count,
-        'bounce_rate': item.bounce_rate,
+        'bounce_rate':  Math.round(item.bounce_rate*100),
```

Comment:
why you changed it back? it was already changed once this week https://github.com/BinaryStudioAcademy/bsa-2019-metrica/commit/e29a82d2024f9a76399eed75951035f5b9c95881#diff-2f4dd8cf22068799777bf0053b7dd492